src/deprecated.jl

```diff
@@ -0,0 +1,24 @@
+
+# BEGIN 1.2.1  deprecations 
+@deprecate get_energy_value(val::BatteryEMS) get_energy_surplus_cost(
```

Comment:
We need a test for this. Please add a new test file for deprecations. `test_deprecations.jl`

Reply:
added the test file